templates/web/site-config.ftl

```diff
@@ -16,6 +16,11 @@
 
 <!doctype html>
 <html xmlns="http://www.w3.org/1999/xhtml">
+
+<#if !(envConfig.role! == 'admin' || envConfig.role! == 'developer')>
+  <script>window.location='/studio/site-dashboard'</script>
+</#if>
```

Comment:
I don't think client-side redirection is sufficient. Being a security adjustment, this still allows viewing the source of the page (e.g. in chrome, you can do `view-source:localhost:8080/studio/site-config`). People could also disable JS, visit the page and reenable and start console manipulations. We should redirect server-side (free marker, groovy controller, whatever). @sumerjabri @dejan-brkic

Reply:
In any case, this should be first line of the head.

Reply:
Here's an enhancement https://github.com/craftercms/studio-ui/pull/1636; Nevertheless, if we had a server-side mechanism we can use now, we should opt for that.